front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -95,10 +96,6 @@ const SimulationResults = ({
     }
   }, [extViewport]);
 
-  if (!trainSimulation) return null;
-
-  if (trainSimulation.status !== 'success') return null;
```

Comment:
This doesn't seem to concern the move of `useSimulationResults` ? It fixes a bug ? This might be better to put it in a different commit. Same for `pathProperties` for the space time chart.

Reply:
You're right :+1:

Reply:
Thank you. Could you add a description in this new commit to explain why we don't make these checks anymore ?

Reply:
I removed these commits and moved them in a new PR: https://github.com/OpenRailAssociation/osrd/pull/9885

Reply:
You didn't keep theses changes after all after the mutation > query refacto ?